Makefile

```diff
@@ -11,7 +11,7 @@ build: vendor
 	go build -o bin/torusblk -ldflags "-X $(REPOPATH).Version=$(VERSION)" ./cmd/torusblk
 
 test: bin/glide
-	go test $(shell ./bin/glide novendor)
```

Comment:
sure. but go default flag can also recognize -- for long flag. we prefer -- over - in our docs. so probably we should also use -- in scripts. but not a big deal.